### PR TITLE
pyats-shell: add page

### PR DESCRIPTION
**Pull request by barweiss45**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/pyats-shell.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# pyats shell
+
+> Starts a pre-loaded pyATS interactive Python Shell to save time in prototyping.
+> More information: <https://pubhub.devnetcloud.com/media/genie-docs/docs/cli/genie_shell.html>.
+
+- Open pyATS shell with defined Testbed file:
```
Comment (kbdharun) [inline pages/common/pyats-shell.md:6]:
```suggestion
- Open pyATS shell with a defined Testbed file:
```

File: pages/common/pyats-shell.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`pyats shell --testbed-file {{/path/to/your/testbed.yaml}}`
+
+- Open pyATS shell with defined Pickle file:
```
Comment (kbdharun) [inline pages/common/pyats-shell.md:10]:
```suggestion
- Open pyATS shell with a defined Pickle file:
```
Usually we don't capitalize in command descriptions but seems like the main documentation follows it too, so I guess it is fine.

File: pages/common/pyats-shell.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# pyats shell
+
+> Starts a pre-loaded pyATS interactive Python Shell to save time in prototyping.
```
Comment (kbdharun) [inline pages/common/pyats-shell.md:3]:
```suggestion
> Start a pre-loaded pyATS interactive Python Shell to save time in prototyping.
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/common/pyats-shell.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://pubhub.devnetcloud.com/media/genie-docs/docs/cli/genie_shell.html>.
+
+- Open pyATS shell with defined Testbed file:
+
+`pyats shell --testbed-file {{/path/to/your/testbed.yaml}}`
```
Comment (EmilySeville7cfg) [inline pages/common/pyats-shell.md:8]:
```suggestion
`pyats shell --testbed-file {{/path/to/testbed.yaml}}`
```

Wrong placeholder syntax is used.

File: pages/common/pyats-shell.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+- Open pyATS shell with defined Pickle file:
+
+`pyats shell --pickle-file {{/path/to/your/pickle.file}}`
```
Comment (EmilySeville7cfg) [inline pages/common/pyats-shell.md:12]:
```suggestion
`pyats shell --pickle-file {{/path/to/pickle.file}}`
```

Wrong placeholder syntax is used.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
- Changes are requested because there are mistakes in the provided page.
- This PR should be reviewed by `pyats-shell` users.

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Nice job. 😄

**Comment by barweiss45** [comment]
Thank you, @kbdharun and @EmilySeville7cfg! Thank you for taking the time to review and provide feedback! 😀👍

**Comment by barweiss45** [comment]
Updated pyats-shell.md with @navarroaxel suggestion. Thank you!

**Pull request merged**